### PR TITLE
Add instruction on how to install tokenizers

### PR DESCRIPTION
**Pull request by thomasw21**

### CODE REVIEW

**Review by stas00 — CHANGES_REQUESTED** [review]
Please merge these 2 sections into the first one. Thank you.

File: jz/envs/README.md [hunk 1]
```diff
@@ -160,13 +160,31 @@ pip install -e .[dev]
 cd $six_ALL_CCFRWORK/code/Megatron-DeepSpeed
 pip install -r requirements.txt
 
+# if needed first:
+# git clone https://github.com/huggingface/tokenizers $six_ALL_CCFRWORK/code/tokenizers
+cd $six_ALL_CCFRWORK/code/tokenizers
+git checkout bigscience_fork
+module load rust
+pip install setuptools_rust
+pip install -e bindings/python
```
Comment (stas00) [inline jz/envs/README.md:169]: sorry, I was trying to say that the `~/.cargo/config.toml` workaround is needed before this line and not somewhere else. 

apologies for not being clear in the first place. would it be easier to just merge this and I will rewrite it?

Comment (thomasw21) [reply jz/envs/README.md:169]: Yeah feel free to modify directly this branch.

**Comment by stas00** [comment]
Thank you, Thomas!

**Pull request merged**